test/src/tests-identity.ts

```diff
@@ -1466,6 +1482,16 @@ describe('identity', function() {
             );
 
         expect(combinedUIsByType).to.have.all.keys([1, 3, 4, 7]);
+        expect(combinedUIsByType[1]).to.equal('customerid1');
+        expect(combinedUIsByType[3]).to.equal('twitter5');
+        expect(combinedUIsByType[4]).to.equal('google4');
+        expect(combinedUIsByType[7]).to.equal('email2@test.com');
+
+        // if an invalid identity type is added to the identity call,
+        // it will be removed from the combinedUIsByType
+        // FIXME: What is the actual test case this is trying to test?
```

Comment:
@rmi22186 We probably should talk about this

Reply:
this is a bizarre test.  i agree. 

there is a `return false` at the bottom of [getIdentityType](https://github.com/mParticle/mparticle-web-sdk/blob/master/src/types.js#L162-L206), which would only ever happen is something that wasn't a real  `identityName` got passed to this.  the idea here is that `newUIByName` has an `invalidKey`, which _would_ get the `return false` except that the logic in the [combineUserIdentities](https://github.com/mParticle/mparticle-web-sdk/blob/master/src/identity.js#L164-L169) already removes this possibility.

does that make sense?

Reply:
Makes sense in a squirrely sort of way.

I would suggest we can get rid of the `expect(combinedUIsByType).to.not.have.property(false)` assertion.

Thoughts?

Reply:
Agreed that it's squirrely.  I think the point of it is to confir that there is a check for the falsey conditional though. Does the test currently fail?  If so i'd say there's a regression somewhere?  it's technically an integration test, and so should pass.  Any reason to not keep it and add commented code for context?